### DISCUSSION TITLE
I’ve installed blink and the extension inside vscode but it doesnt show the folders in vscode. And it doesnt create a blink folderon my ipad either. Can you hel me?

### DISCUSSION THREAD

**Discussion by AriCohelo** [blinksh/blink#1706] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

**Comment by carloscabanero** [comment]
Hi! Can you give us more information of what you are seeing with screenshots? What blink folder do you mean?

**Reply by AriCohelo** [reply]
For some reason i’ve entered now to make screenshots and tree of folders is showing now, but i couldn’t make it show before. I suppose it solutioned itself. Ím gonna try to work and update if i have trubles with it.
Thanks
![6EC68053-2969-431D-8724-8F654121D913](https://user-images.githubusercontent.com/125672519/220452766-48ce886e-8517-4ae5-9123-522c4ed06477.png)

**Comment by carloscabanero** [comment]
Sometimes Code just needs a restart to reload the extensions. You should be good now.

**Comment by AriCohelo** [comment]
Where are blink files located on my ipad, in order to drop files for a project?

**Reply by carloscabanero** [reply]
You have access to the Documents folder from the Files.app. If you don't see a Blink folder there, do a `touch Documents/test` from a shell, and that should enable the directory.

**Comment by AriCohelo** [comment]
Excelent my frind, that worked. Than you very much

**Comment by AriCohelo** [comment]
EXcuseme for so many questions but i’m super new in this. I want to create a file inside the documents folder ad when I try to do it it creates the file in the Home folder. How can I create the file in the place I want. Or if there is a wayto move it i didn’t fiind that.

**Reply by carloscabanero** [reply]
No worries. To create in a a specific folder, you need to have that folder selected and just tap the file+ icon button. If this is your first time using VS Code, I recommend you to watch https://www.youtube.com/watch?v=B-s71n0dHUk. That may help a lot.

**Reply by AriCohelo** [reply]
I Think i’m gonna make good use of this video thanks. But what you describe is exactly what i’m doing. I restarted the app just in case but it happens again

**Reply by AriCohelo** [reply]
I’ve achieved that selecting a file inside the folder i want the new file.
